server/database/seeders/index.js

```diff
@@ -1,29 +1,15 @@
-var path = require("path"),
-  fs = require("fs"),
-  db = require("../index");
+/* eslint-disable no-console */
+const filesReader = require("../../helpers/readFiles");
+const db = require("../index");
 
-const seeds = {};
-const basename = path.basename(__filename);
-String.prototype.capitalize = function() {
-  return this && this[0].toUpperCase() + this.slice(1);
-};
-fs.readdirSync(__dirname)
-  .filter(file => {
-    return (
-      file.indexOf(".") !== 0 && file !== basename && file.slice(-3) === ".js"
-    );
-  })
-  .forEach(file => {
-    const seed = require(path.join(__dirname, file));
-    seeds[file.replace(".js", "").capitalize()] = seed;
-  });
+const seeds = filesReader(__dirname, __filename);
 
 (async () => {
   const promises = Object.keys(seeds).map(async seedName => {
-    console.log("seeding " + seedName + "...");
-    return await db
+    console.log(`seeding ${seedName}...`);
+    await db
       .query(...seeds[seedName])
-      .then(() => console.log(seedName + " seeded successfully"));
+      .then(() => console.log(`${seedName} seeded successfully`));
```

Comment:
why are you console logging here?

Reply:
Just to know that my seeders are passing successfully, but I removed them now